training/training.module

```diff
@@ -0,0 +1,378 @@
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/menu/callback'] = array(
+    'title' => 'Callback Menu Item',
+    'page callback' => 'training_menu_callback',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+    'type' => MENU_CALLBACK,
+  );
+
+  $menu['training/menu/tabs'] = array(
+    'title' => 'Tabs Pages',
+    'page callback' => 'training_menu_tabs_default',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/menu/tabs/first'] = array(
+    'title' => 'Tabs',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+    'weight' => -1,
+    'type' => MENU_DEFAULT_LOCAL_TASK,
+  );
+  $menu['training/menu/tabs/second'] = array(
+    'title' => 'Second',
+    'page callback' => 'training_menu_tabs',
+    'page arguments' => array(3, ),
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+    'weight' => 0,
+    'type' => MENU_LOCAL_TASK,
+  );
+  $menu['training/menu/tabs/third'] = array(
+    'title' => 'Third',
+    'page callback' => 'training_menu_tabs',
+    'page arguments' => array(3, ),
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+    'weight' => 1,
+    'type' => MENU_LOCAL_TASK,
+  );
+  $menu['training/menu/tabs/fourth'] = array(
+    'title' => 'Fourth',
+    'page callback' => 'training_menu_tabs',
+    'page arguments' => array(3, ),
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+    'weight' => 2,
+    'type' => MENU_LOCAL_TASK,
+  );
+  $menu['training/menu/tabs/default/first'] = array(
+    'title' => 'Default secondary tab',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+    'tab_parent' => 'training/menu/tabs/first',
+    'type' => MENU_DEFAULT_LOCAL_TASK,
+  );
+  $menu['training/menu/tabs/default/second'] = array(
+    'title' => 'second',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+    'tab_parent' => 'training/menu/tabs/first',
+    'type' => MENU_LOCAL_TASK,
+  );
+  $menu['training/menu/tabs/default/third'] = array(
+    'title' => 'third',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+    'tab_parent' => 'training/menu/tabs/first',
+    'type' => MENU_LOCAL_TASK,
+  );
+  $menu['training/menu/arguments/%/%'] = array(
+    'title' => 'Menu arguments',
+    'page callback' => 'training_menu_arguments',
+    'page arguments' => array(3,4 ),
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/menu/placeholder/%/display'] = array(
+    'title' => 'Menu argument before display',
+    'page callback' => 'training_menu_arguments_display',
+    'page arguments' => array(3 ),
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/menu/default_arg/%digit'] = array(
+    'title' => 'Menu preprocessing argument',
+    'page callback' => 'training_menu_arguments_preprocessing',
+    'page arguments' => array(3 ),
```

Comment:
Arrays should be formatted with a space separating each element (after the comma)